share/dkg/pedersen/dkg.go

```diff
@@ -222,12 +243,14 @@ func NewDistKeyHandler(c *Config) (*DistKeyGenerator, error) {
 
 // NewDistKeyGenerator returns a dist key generator ready to create a fresh
 // distributed key with the regular DKG protocol.
-func NewDistKeyGenerator(suite Suite, longterm kyber.Scalar, participants []kyber.Point, t int) (*DistKeyGenerator, error) {
+func NewDistKeyGenerator(suite Suite, longterm kyber.Scalar, participants []kyber.Point, t int, r io.Reader, userOnly bool) (*DistKeyGenerator, error) {
```

Comment:
Let's keep the API as it is for now, we don't need to make API-breaking changes for this. This function will use the default values.